.gitignore

```diff
@@ -1,9 +1,7 @@
-.idea
+.idea/
 *.iml
-.gradle
-.gradletasknamecache
-build
-out
-target
-local.properties
```

Comment:
I would suggest keeping the `local.properties` entry here as this file may be useful for overriding some KGP properties locally, e.g. for experiments or manual testing.

Reply:
Good idea, thanks. I assumed that local.properties was only for Android development.